src/main/scala/geekie/mapreddemo/AkkaMapreduceBenchmark.scala

```diff
@@ -1,62 +1,40 @@
 package geekie.mapreddemo
```

Comment:
Por que que eu to recebendo esse email? :-P

On Mon, Jul 20, 2015 at 2:59 PM rbaron <notifications@github.com> wrote:

> In src/main/scala/geekie/mapreddemo/AkkaMapreduceBenchmark.scala
> <https://github.com/projetoeureka/akka-mapreduce/pull/2#discussion_r35023716>
> :
>
> > @@ -1,62 +1,40 @@
> >  package geekie.mapreddemo
>
> Maybe geekie.mapred.examples?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projetoeureka/akka-mapreduce/pull/2/files#r35023716>.
>